atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -221,6 +221,14 @@ def set_operators_metadata(self, extra, worker_metadatas):
 
                     return  # only one worker can process operator manifests
 
+    def set_pnc_build_metadata(self, extra):
+        plugin_results = self.workflow.postbuild_results.get(PLUGIN_GENERATE_MAVEN_METADATA_KEY) \
+                         or {}
+        pnc_build_ids = plugin_results.get('pnc_build_ids')
+
+        if pnc_build_ids:
+            extra['image']['pnc_build_ids'] = pnc_build_ids
```

Comment:
I'd prefer `['pnc']['build_ids']`, namespacing for future extensions

Reply:
or even
```
'pnc' : {
    'builds': [
        {'id': 123},
    ]
}
```

if you want to store metadata per PNC build in future

Reply:
yeah good idea

Reply:
Idea applied :)